pkg/services/star/model.go

```diff
@@ -2,7 +2,10 @@ package star
 
 import "errors"
 
-var ErrCommandValidationFailed = errors.New("command missing required fields")
+var (
+	ErrCommandValidationFailed = errors.New("command missing required fields")
+	ErrDashboardAlreadyStarred = errors.New("dashboard was already starred")
```

Comment:
nit
I would suggest declaring this as `github.com/grafana/grafana/pkg/util/errutil.Error` as suggested [here](https://github.com/grafana/grafana/blob/main/contribute/backend/errors.md#declaring-errors)